common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -0,0 +1,33 @@
+﻿@maa_01 = "gfx/interface/icons/culture_innovations/innovation_maa_01.dds"
```

Comment:
Time to move to the next stage, modifiers.
We've added new unit types, but they're not buffed anywhere like buildings, character traits and perks.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103153675-afd6c600-47ab-11eb-81ac-62b419e612c7.png)
![image](https://user-images.githubusercontent.com/46384992/103153681-bbc28800-47ab-11eb-86c2-b296f7c2403f.png)
![image](https://user-images.githubusercontent.com/46384992/103153693-ce3cc180-47ab-11eb-977b-269fa150b419.png)

</details>

Like why Hit and Run buffs heavy infantry, but not giants?

Thus you should look through all vanilla unit type bonuses and maluses and add new unit types there as well.

Reply:
won't we be using a new skill tree?

Reply:
ill look into buildings

Reply:
> won't we be using a new skill tree?

No? I haven't heard such planes.